draft-packages/form/KaizenDraft/Form/TextAreaField/README.mdx

```diff
@@ -0,0 +1,151 @@
+
+## Copy guidelines
+
+To be scannable and fully helpful, a text area must have an associated label. Here’s what you should know:
+
+### Give it a clear, short label
+
+Make sure your text area has a short, descriptive label above it that describes what the user should type into the box below. Avoid phrasing your labels as questions. Keep in mind that your label shouldn’t contain instructions. Reserve those for the helper text.
+
+<DoAndDontContainer>
+<Do>
+
+- Comment
+
+</Do>
+
+<Dont>
+
+- Enter your comment here
+
+</Dont>
+</DoAndDontContainer>
+
+### Placeholder text should be examples and not instructions
+
+Make sure that any text inside the field box serves as an example of what the user should type into the box. Don’t put any instructions inside the box. 
+
+<DoAndDontContainer>
+<Do>
+
+- e.g. Thanks for everything!
+
+</Do>
+
+<Dont>
+
+- Thanks for everything!
+
+</Dont>
+</DoAndDontContainer>
+
+Keep in mind that examples aren’t always required. For example, a text area asking for someone’s opinion doesn’t need an example in it if it will be misleading. Only use placeholder text when required. More often than not, blank text fields are easier to spot and scan.
+
+### Error text should be succinct
+
+Keep all error text short, useful and to the point.
+
+<DoAndDontContainer>
+<Do>
+
+- You must enter a comment
+
+</Do>
+
+<Dont>
+
+- You entered a comment that is invalid
+
+</Dont>
+</DoAndDontContainer>
+
+### Helper text should provide extra guidance
+
+Helper text placed below the field can include further instructions or information on how to fill out the text area. It can also include details on what will happen with the filled out information. Don’t put placeholders here.
+
+<DoAndDontContainer>
+<Do>
+
+- Your comment will be used anonymously
+
+</Do>
+
+<Dont>
+
+- For example, “Thanks for everything!”
+
+</Dont>
+</DoAndDontContainer>
+
+### Write in sentence case with minimal punctuation
+
+All text area copy should be in sentence case i.e. the first letter of the first word is capitalized and everything else is in lower case unless it’s a proper noun or feature name. Always err on the side of minimal punctuation, so leave out full stops and commas where possible.
+
+## To keep in mind
+
+* **Optional vs required:**
+    * As a rule of thumb, generally use “ (optional)” to indicate optional fields instead of “`*`” to indicate required fields.
+* **Placeholder text** is temporary and vanishes once a user enters text. Avoid placeholder text that must be deleted by the user. Avoid using placeholder text as a label as the user will lose the context of what field they're filling out as soon as it disappears.
+* For **sizing**, consider likely text input and use existing data where possible.
+* For **right-to-left (RTL)** languages, the layout of a text area is swapped. The validation icons are left-aligned.
+
+### Validation
```

Comment:
Do we have guidelines for validation that we could link here? It shouldn't live on this page, but could be useful to link to - when to use negative vs. caution, etc.